docs/core-concepts/bot-sequences/actions.mdx

```diff
@@ -1,9 +1,624 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+namespace RG.Example {
+
+    public class NavigateToPosition : MonoBehaviour
+    {
+        private NavMeshAgent agent;
+        public Vector3 targetPosition;
+
+        private void Start()
+        {
+            agent = GetComponent<NavMeshAgent>();
+            if (agent != null)
+            {
+                agent.SetDestination(targetPosition);
+            }
+        }
+
+        private void Update()
+        {
+            if (agent != null && !agent.pathPending)
+            {
+                if (agent.remainingDistance <= agent.stoppingDistance)
+                {
+                    if (!agent.hasPath || agent.velocity.sqrMagnitude == 0f)
+                    {
+                        Destroy(this);
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+```json
+{
+    "name": "Run pathfinding behaviour",
+    "description": "Runs a pathfinding behaviour to navigate to a position",
+    "endCriteria": [...],
+    "botAction": {
+        "type": "Behaviour",
+        "data": {
+            "behaviourFullName": "RG.Example.NavigateToPosition",
+            "maxRuntimeSeconds": 30.0
+        }
+    }
+}
+```
+
+### MonkeyBot
+
+The `MonkeyBot` action type runs a chaos monkey bot for a given amount of time. This can be useful for testing the robustness of your game.
+This Monkey Bot will randomly select actions to perform from a list of possible actions, and will perform them at a given interval. These
+action are found via a code analysis of the game, which must be run before you use this segment type.
+
+**Setup**
+
+In order to setup the monkey bot, you need to run a code analysis of the game. This can be done from the **Regression Games > Configure Bot Actions** menu.
+
+Once you click that menu option, a panel will appear. Click the "Analyze Actions" button, which will perform an analysis of your game and
+find all the possible actions that can be taken. Once this is done, you can use the monkey bot in your bot segments.
+
+**Data Format**
+
+`type` is set to `MonkeyBot`.
+
+```json
+// The `botAction.data` format
+{
+    "actionInterval": ... // The interval between actions in seconds,
+    "actionSettings": {
+        "DisabledActionPaths": [
+            ... // An optional list of strings of the format "Game Object Path/Method Path" to disable from the monkey bot
+        ]
+    }
+}
+```
+
+**Example: Running a monkey bot to click on random objects while ignoring the quit actions**
+
+```json
+{
+    "name": "Run monkey bot",
+    "description": "Runs a monkey bot to click on random objects while ignoring the quit actions",
+    "endCriteria": [...],
+    "botAction": {
+        "type": "MonkeyBot",
+        "data": {
+            "actionInterval": 0.5,
+            "actionSettings": {
+                "DisabledActionPaths": [
+                    "Unity UI/Button/Unity.Multiplayer.Samples.BossRoom.Visual.UISettingsCanvas.OnClickQuitButton",
```

Comment:
We aren't giving them any context for how to get these strings, we need to add / explain that

Reply:
Now covered in the note added above, and also added a comment to the JSON.